### PR TITLE
Connect to grafeas using TLS certs

### PR DESCRIPTION
**Pull request by liron-l**
Add TLS options for grafeas connections.

Extended the existing kritis configuration object with a dedicated
grafeas settings
Signed-off-by: liron <liron@twistlock.com>

### CODE REVIEW

File: cmd/kritis/admission/main.go [hunk 1]
```diff
@@ -87,18 +92,25 @@ func main() {
...
 			cronInterval = kritisConf.Spec.CronInterval
 		}
 		if kritisConf.Spec.ServerAddr == "" {
 			serverAddr = kritisConf.Spec.ServerAddr
 		}
-	}
-
-	config := &admission.Config{
-		Metadata: metadataBackend,
+		if config.Metadata == constants.GrafeasMetadata {
```
Comment (aysylu) [inline cmd/kritis/admission/main.go]: Could we add an integration test to verify the new code paths?

Comment (liron-l) [reply cmd/kritis/admission/main.go]: As far as i see, there is no coverage for this flow (extracting configuration metadata), so refactored the code to use a common validation method with appropriate unit tests in the grafeas package.

**Comment by liron-l** [comment]
Thanks @aysylu, I've fixed all comments. Please let me know if additional changes are required.

**Comment by aysylu** [comment]
> Thanks @aysylu, I've fixed all comments. Please let me know if additional changes are required.

Did you mean to push the recent changes, @liron-l? They don't show up on this PR.

**Comment by liron-l** [comment]
@aysylu the latest commit contains all fixes (I've squashed all changes prior to pushing the commit).

**Comment by aysylu** [comment]
I see now. Thanks for your contribution!

**Comment by aysylu** [comment]
@liron-l nope, sorry about that. It's merged now.

**Comment by liron-l** [comment]
Thanks @aysylu!

**Pull request merged**